itest/starboard_cli_test.go

```diff
@@ -710,8 +718,76 @@ var _ = Describe("Starboard CLI", func() {
 
 	})
 
-	PDescribe("Command audit configuration", func() {
-		// TODO Test polaris command
+	Describe("Command polaris configuration", func() {
+		// containerNameAsIDFn is used as an identifier by the MatchAllElements matcher
+		// to group ConfigAuditReport by container name.
+		containerNameAsIDFn := func(element interface{}) string {
```

Comment:
Actually you're using resource name label to group items. So this method should be called `resourceNameAsIDFn`.
BTW, wee only use container name labels for Vulnerability reports. For ConfigAuditReports, such report contains checks for the Pod template and its containers.